components/eamxx/src/physics/share/physics_constants.hpp

```diff
@@ -126,6 +126,29 @@ struct Constants
   static constexpr Scalar earth_ellipsoid3 = 1.175;     // third expansion coefficient for WGS84 ellipsoid
 };
 
+template <typename Scalar>
+struct P3_Constants
+{
+
+private:
+   static Scalar p3_autoconversion_factor;
+
+public:
+static void set_p3_defaults(){
+  set_p3_autoconversion_factor(1350.0);
+};
+
+static Scalar get_p3_autoconversion_factor(){
```

Comment:
Btw, there is no difference between having `p3_autoconversion_factor` public and having it private with get/set methods. If set/get are to be provided (and they don't perform any side-effect work, but simply set/get the value of the member), then it is much simpler to make the parameter a public member of the class.